src/client/components/AddNewRoom/AddNewRoomStyle.css

```diff
@@ -0,0 +1,60 @@
+* {
```

Comment:
Do not add this global style here. We have `index.css` in the client folder for global styles.